### DISCUSSION TITLE
Minimum distance from wall?

### DISCUSSION THREAD

**Discussion by justinj57** [awawa-dev/HyperHDR#685] [General]
My TV is on a wall, but angled such that the top is about 8 inches from the wall and the bottom is only an inch or so from the wall. I'm building a 45 degree frame to mount the LEDs on. Is there a minimum distance I should mount them from the wall, or can I place the 45 degree from directly on the wall?

**Comment by thomasadelhardt** [comment]
I have mounted my led strip on a 45 degrees angled wooden frame, it's about 5 cm (2 inches) from the wall on the top and the sides.
That look great but unfortunately my TV has a weird shape so the frame at the bottom is about 4 mm (1/8 inch) from the wall and that is to close even when its angled at 45 degrees. I'm guessing that if the leds could be at angled so it is pointing more downward it would look better.
